fsm/leave_add_csync.go

```
@@ -276,9 +288,8 @@ func LeaveVerifyCsyncPublished(z *music.Zone) bool {
 		return false
 	}
 
-	// compare that the csync records are the same
+	// compare that the CSYNC records are the same as the created CSYNC
 	for _, csyncrr := range csynclist {
-		//if csyncrr.String() != z.CSYNC.String() {
 		if !dns.IsDuplicate(csyncrr, z.CSYNC) {
 			z.SetStopReason(fmt.Sprintf("CSYNC records are not identical"))
 			return false
```

Same here, this function should probably return an error instead of bool.